app/infra/models/memoir.py

```diff
@@ -15,5 +16,6 @@ def to_json(self, *args, **kwargs):
             "creator": str(self.creator),
             "text": str(self.text),
             "creation_date": str(self.creation_date),
-            "last_updated": str(self.last_updated)
+            "last_updated": str(self.last_updated),
+            "media_url": str(self.media_url)
```

Comment:
these should be `camelCased` so that it's easier to access in javascript. if it's `snake_cased` it's a little harder

Reply:
only in the `.to_json()` methods do we use camelCasing, all the internal only python code is snake_case